Data/Sys/GameSettings/G3L.ini

```diff
@@ -1,4 +1,4 @@
-# G3LE8P - Super Monkey Ball Adventures (TM)
+# G3LE8P - Super Monkey Ball Adventure
```

Comment:
Titling in the ini files was done automatic in the past from dolphin, and many such cases remain. Since it isn't used anywhere outside of the gameinis i had little incentive to care much about titles.